tests/integration/tier1/convert-offline-systems/prepare_system.py

```diff
@@ -47,10 +47,6 @@ def test_prepare_system(shell):
 
     configure_connection()
 
-    # c2r checks the internet connectivity against the 8.8.8.8 DNS server
-    # pinging specific IP address is still possible so we need to block this as well
-    assert shell("iptables -I OUTPUT -d 8.8.8.8 -j DROP").returncode == 0
```

Comment:
IIRC, this was mainly placed here because of PR#458 where we were checking against google dns, since we are using an address now, I think this is not required anymore.

Reply:
Yes that should not be needed anymore.